index.js

```diff
@@ -3,10 +3,8 @@
 const bunyan = require('bunyan');
 const raven = require('raven');
 const sentryStream = require('bunyan-sentry-stream');
-const logstashStream = require('bunyan-logstash-tcp');
```

Comment:
Please also remove the dependency from the package.json

Reply:
Done